jobs/e2e_node/containerd/containerd-master/image-config.yaml

```diff
@@ -4,6 +4,6 @@ images:
     project: ubuntu-os-gke-cloud
     metadata: "user-data</go/src/github.com/containerd/cri/test/e2e_node/init.yaml,containerd-configure-sh</go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env</workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env"
   cos-stable:
-    image: cos-77-12371-274-0
+    image: cos-81-12871-1185-0
```

Comment:
why not `image_family` here as well with the `lts`?

Reply:
https://github.com/kubernetes/test-infra/pull/17734#discussion_r469582242

Reply:
🤣  ok. This comment was before we discussed it on Monday. Now I feel that consistently make it `81-lts` would be better.